json-rpc/API-CHANGELOG.md

```diff
@@ -13,6 +13,22 @@ Please add the API change in the following format:
 
 ```
 
+## 2021-06-08 Add `get_account_transactions_with_proofs` API
+
+This new API allows verifying client users to get transactions under an account
+along with the corresponding proofs needed to verify the response.
+
+## 2021-06-08 Add optional `ledger_version` parameter to `get_account_transaction(s)` methods
```

Comment:
I don't think this is needed, it is confusing for client to specify the ledger_version when requesting account txns. API should not be changed only because the test needs it. I think your test can filter result to compare verifying and non-verifying client results.

Reply:
sounds good, removed the extra parameters